packages/ember-htmlbars/lib/helpers/if_unless.js

```diff
@@ -7,6 +7,42 @@ import Ember from "ember-metal/core"; // Ember.assert
 import shouldDisplay from "ember-views/streams/should_display";
 
 /**
+  Use the `if` block helper to conditionally render a block depeneding on a property. If
+  the property is "falsey", for example: `false`, `undefined` `null`, `""`, `0` or
+  an empty array, the block will not be rendered.
+  ```handlebars
+  // will not render if foo is falsey
+  {{#if foo}}
+    Welcome to the {{foo.bar}}
+  {{/if}}
+  ```
+  You can also specify a template to show if the property is falsey by using the `else` helper.
+  ```handlebars
+  //Is it raining outside?
+  {{#if isRaining}}
+    Yes, grab an umbrella!
+  {{else}}
+    No, it's lovely outside!
+  {{/if}}
+  ```
+  You are also able to combine `else` and `if` helpers to create more complex conditional logic.
+  ```handlebars
+  {{#if isMorning}}
+    Good morning
+  {{else if isAfternoon}}
+    Good afternoon
+  {{else}}
+    Good night
+  {{/if}}
+  ```
+  You can use `if` inline to conditionally render a single property or string.
+  ```handlebars
+  {{if useLongGreeting "Hello" "Hi"}} Dave
```

Comment:
should probably clarify that this is a ternary, and what the three arguments are.